E/Exodus/Deps.toml

```diff
@@ -0,0 +1,7 @@
+[0]
+BinaryBuilder = "12aac903-9f7c-5d81-afc2-d9565ea332ae"
```

Comment:
I don't think depending on BinaryBuilder is correct, you don't use it for anything. No package should depend on BinaryBuilder

Reply:
my fault I had stupidity in my toml file